src/rt/typeinfo/ti_n.d

```diff
@@ -21,7 +21,7 @@ class TypeInfo_n : TypeInfo
 
     override size_t getHash(scope const void* p) const
     {
-        return 0;
+        return hashOf(cast(void*)null);
```

Comment:
Post https://github.com/dlang/druntime/pull/2246 this should be the same as `return 0`.

Reply:
Same above:
`hashOf([null, null])` and `hashOf([null, null, null])` should have different values, thus we need chaining by seed, thus we need to rehash seed 0 each time.